### DISCUSSION TITLE
Does Zephyr supports CRUN runtime and can we run containers on Zephyr

### DISCUSSION THREAD

**Discussion by Franky45** [zephyrproject-rtos/zephyr#73137] [Q&A]
Hi,
Can we run containers on Zephyr?. Does Zephyr supports CRUN runtime

**Comment by jukkar** [comment]
No, zephyr is not linux and is meant for embedded devices, so it does not have container support.

**Comment by Franky45** [comment]
but as per some research i get to know that zephyr rtos supports ocre/atym container runtime is it true

**Reply by jukkar** [reply]
This was new to me, but according to this https://zephyrproject.org/ocre-a-tiny-open-source-container-runtime-for-embedded-systems/ the Atym seems to support this. I have not seen anything in upstream zephyr for this.
